sanssouci/lambda_calibration.py

```diff
@@ -50,30 +50,29 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
     """
 
     # Init
-    n = X.shape[0]
-    p = X.shape[1]
+    n, p = X.shape
 
     # Step 1: calculate $p$-values for B permutations of the class assignments
 
     # 1.1: Intialise all vectors and matrices
-    shuffled_categ_current = categ.copy()
+    labels_shuffled = labels.copy()
 
-    shuffled_categ_all = np.zeros([B, n])
+    labels_shuffled_all = np.zeros((B, n))
```

Comment:
```suggestion
    all_shuffled_labels = np.zeros((B, n))
```

Reply:
fixed in c2fd1a3ed906d58ba57807a78aa25e7237ccd87e